week1/homework/src/server.js

```diff
@@ -9,7 +9,40 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    const reqUrl = request.url;
+    if (reqUrl === '/' || reqUrl === '/state') {
+      response.writeHead(200, { 'Content-Type': 'application/json' });
+      const home = {
+        'state': state
+      };
+      response.end(JSON.stringify(home, null, 2));
+    } else if (reqUrl === '/add') {
+      response.writeHead(200, { 'Content-Type': 'application/json' });
+      const add = {
+        'state': ++state
+      };
+      response.end(JSON.stringify(add, null, 2));
+    } else if (reqUrl === '/subtract') {
```

Comment:
It's OK to use `if` and `else if`, but at some point it could ben more readable (and less code) to use a `switch` with different `case`s.

Reply:
I'll fix my code, thanks a lot for the review👍

Reply:
Fixed the code, would you check it after the changes that I made @J05J4